src/Domain.LinnApps/ImportBooks/ImportBook.cs

```diff
@@ -122,5 +122,8 @@ public class ImportBook
         public IList<ImportBookOrderDetail> OrderDetails { get; set; }
 
         public IList<ImportBookPostEntry> PostEntries { get; set; }
+
+        public Supplier FullSupplier { get; set; }
+        public Supplier FullCarrier { get; set; }
```

Comment:
add blank line? What does fullsupplier or fullcarrier mean out of interest

Reply:
Just that it's the full supplier object which isn't normally on an impbook, it only stores supplierId. Maybe I should just call them Supplier and Carrier? For some reason I wanted to differentiate, I also only include/pull them in for filterBy and not findById

Reply:
yeah i'd call them supplier and carrier.